xonsh/proc.py

```diff
@@ -1408,7 +1410,7 @@ def wait(self, timeout=None):
 
     def _signal_int(self, signum, frame):
         """Signal handler for SIGINT - Ctrl+C may have been pressed."""
-        # check if we have already be interrupted to prevent infintie recurrsion
+        # check if we have already be interrupted to prevent infinite recursion
```

Comment:
At least this is a grammatical error and not another spelling error 😄 

I'll fix this too!